src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/SupplierAssessment.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity
+
+import org.hibernate.annotations.Fetch
+import org.hibernate.annotations.FetchMode
+import java.util.UUID
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.OneToMany
+import javax.persistence.OneToOne
+import javax.persistence.Table
+import javax.validation.constraints.NotNull
+
+@Entity
+@Table(name = "supplier_assessment")
+data class SupplierAssessment(
+  @Id val id: UUID,
+  @NotNull @OneToOne val referral: Referral,
+
+  @NotNull @OneToMany @Fetch(FetchMode.JOIN) val appointments: MutableSet<Appointment> = mutableSetOf(),
+) {
+  val appointment: Appointment
```

Comment:
please can we name this simliarly to the action plan one `currentAppointment`? one of the intentions with that was to make a future find & replace easier when we actually have the possibilty there are multiple appointments per thing.